js/engine/Main.ml

```diff
@@ -48,15 +48,8 @@ let _ =
            in
            let targets =
              List.map
-               (fun f ->
-                 `CodeTarget
-                   Input_to_core_t.
-                     {
-                       path = f;
-                       analyzer = xlang;
-                       products = Product.all;
-                       lockfile_target = None;
-                     })
+               (fun f : Target.t ->
+                 Code (Target.mk_code xlang Product.all (File (Fpath.v f))))
```

Comment:
I still think 'code' is bad. I'd rather have mk_regular_target and mk_lockfile_target.
cc @mmcqd for ideas for better names for this.

Reply:
"regular target" and "lockfile target" sound good to me

Reply:
I renamed this to `mk_regular` as part of the change in e18e0a6b4e5b7776a80ac46ff158b1490806f156.

Given these are always used like `Target.mk_regular` or `Target.mk_lockfile` I don't see the point of including `target` again in the function name (similar to `Target.path` being preferred to `Target.target_path`).